packages/webview_flutter/lib/src/webview_ios.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import '../platform_interface.dart';
+import 'webview_method_channel.dart';
+
+class WebViewIosImplementation implements WebViewPlatformInterface {
```

Comment:
Avoid using `Ios` in classes, the correct capitalization is `iOS`. Use `Cupertino` if necessary.

Reply:
done